index.js

```diff
@@ -1,21 +1,150 @@
 'use strict';
 const {app, BrowserWindow} = require('electron');
+const bufferToArrayBuffer = require('buffer-to-arraybuffer');
 
-module.exports = async (url, {type = 'json', ...options} = {}) => {
+let win;
```

Comment:
```suggestion
let window_;
```

Reply:
In that cases I like to use `$window`, what do you think? Anyway, I already comitted `window_`.

Reply:
I prefer `window_`. It's what I'm using in other projects when a name is already taken.